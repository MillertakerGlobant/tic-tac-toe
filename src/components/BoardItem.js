import React, { useState, useEffect } from 'react';

const BoardItem = () => {
  return (
    <div>
      Game Board Item 
    </div>
  )
}

export default BoardItem;