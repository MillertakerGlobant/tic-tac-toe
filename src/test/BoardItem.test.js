import { shallow } from 'enzyme';
import BoardItem from '../components/BoardItem';

describe('<BoardItem />  Component test', () => {
  var wrapper = shallow(<BoardItem />);
  beforeEach(() => {
    wrapper = shallow(<BoardItem />);
  });

  test('renders empty BoardItem component', () => {
    expect(wrapper).toMatchSnapshot();
  });
})

