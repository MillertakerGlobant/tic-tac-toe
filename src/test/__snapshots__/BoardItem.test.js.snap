// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BoardItem />  Component test renders empty BoardItem component 1`] = `
<div>
  Game Board Item
</div>
`;
